css/components/_past.scss

```diff
@@ -0,0 +1,60 @@
+.past {
+  padding: $basic-padding;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+
+  & hr {
+    width: 30px;
+    margin: 0;
+  }
+
+  & .list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 15px;
+    width: 100%;
+    margin: 24px 0;
+
+    @include desktop {
+      flex-direction: row;
+    }
+
+    & .card {
+      background-size: cover;
+      border-radius: 5px;
+      color: #fff;
+      font-weight: bold;
+      width: 100%;
+      max-width: 400px;
+      // aspect ratio
+      // aspect-ratio: 1.5%;
+
+      &:first-child {
+        background-image: url(../images/2015.jpg);
+      }
+
+      &:last-child {
+        background-image: url(../images/2013.png);
+      }
+
+      & .content {
+        border-radius: 5px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        background-color: transparentize($primary, 0.2);
+        width: 100%;
+        height: 100%;
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, 
### The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
Please fix your mobile design of the Past Youth Weeks section; that the transparent orange cover may cover the photos fully to match the original design.
### Your Design
![twa1](https://user-images.githubusercontent.com/63964646/148644164-f90140f0-4ac9-44bc-a6a7-682a764b1c49.PNG)

### Original Design
![twa2](https://user-images.githubusercontent.com/63964646/148644169-e4ac799f-0e48-4ac3-95d1-89e0822b65f4.PNG)

Reply:
![Screenshot (64)](https://user-images.githubusercontent.com/37457094/148644954-e8966ec2-62fd-4de7-acaf-50249db9682e.png)

Reply:
This is what I
 have here, can you enlight me more on it

Reply:
@ismailakinkunmi they are referring to your mobile version. your desktop version is fine.

Reply:
![Screenshot (65)](https://user-images.githubusercontent.com/37457094/148651271-95df9b90-eafa-4b37-8bff-a1cc3ddfb350.png)

Reply:
desktop are side by side while mobile is on top of one another, please make the feedback clear or maybe we should both look into it together.

Reply:
@ismailakinkunmi Hi, you are showing me the desktop version. we are referring to the **mobile** version. 
Please put your screen on mobile version and look at the red background of the image. the red background doesn't overlay the full image on your mobile version it is visible only on a small piece of the top of the image. 
## Your Design (On Mobile)
![image](https://user-images.githubusercontent.com/18730106/148651681-60bd8144-9257-4acc-9e67-800d639a14a9.png)